lifecycle/lifecycle.go

```diff
@@ -221,7 +247,10 @@ func PublishEvent(e Event, conn inter.RawNATSConnector) error {
 		return err
 	}
 
-	conn.PublishRaw(target, j)
+	err = conn.PublishRaw(target, j)
+	if err != nil {
+		return err
```

Comment:
Not sure where we are calling this - but generally publishing events is best efforts so I specifically didnt return eror here, but I guess its harmless to do so

Reply:
ah